methods/__init__.py

```diff
@@ -0,0 +1,23 @@
+def get_modules():
+    from os.path import dirname, basename, isfile, join
+    import glob
+    import importlib
+    
+    return [importlib.import_module("methods.%s" % basename(f)[:-3])
+            for f in glob.glob(join(dirname(__file__), "*.py"))
+            if isfile(f) and basename(f) != "__init__.py"]
+
+def get_functions():
+    from os.path import dirname, basename, isfile, join
+    import glob
+    import importlib
+  
+    return dict([func_key_value 
+            for module in get_modules()
```

Comment:
there's some of [x for x in [y for y in [z for z in [1,2,3]]], which can just collapse to [1,2,3] here. Could be (i think):
```
    return dict([                                                                                                                                    
        [method, getattr(module, method)]                                                                                                            
        for module in get_modules()                                                                                                                  
        for method in dir(module)                                                                                                                    
        if (getattr(getattr(module, method)), '__name__', None) is "pythia_func"                                                                     
    ])
```